openstack/identity/v3/projects/requests.go

```diff
@@ -243,3 +243,30 @@ func Update(ctx context.Context, client *gophercloud.ServiceClient, id string, o
 	_, r.Header, r.Err = gophercloud.ParseResponse(resp, err)
 	return
 }
+
+// GetProjectTag checks if a a tag exists in a project.
+func GetProjectTag(ctx context.Context, client *gophercloud.ServiceClient, projectID string, tag string) (r GetProjectTagResult) {
```

Comment:
We're already in the project package, there's no need to have Project in the name of the function. Also, we should be more descriptive with the name. Perhaps call it `CheckTag()`?